docs/Release_Guide/release_steps/update_release_notes_development.rst

```diff
@@ -0,0 +1,24 @@
+Update Release Notes
+--------------------
+
+You can refer to the GitHub Issues page to see what has changed for this
+release. Open the following URL in a browser:
+
+.. parsed-literal::
+
+    https://github.com/dtcenter/|projectRepo|/issues
+
+* Click on the Projects tab and select the project (under Repository) that
+  corresponds to the release you are creating.
+
+* Update the release-notes.rst file found in the User's Guide directory.
+
+* If you are creating a beta1 release, remove the previous version's release
```

Comment:
This same instruction is listed twice.

Reply:
Thanks for catching that. I just fixed it up.